plugin/trino-raptor-legacy/src/main/java/io/trino/plugin/raptor/legacy/metadata/DatabaseMetadataModule.java

```diff
@@ -85,7 +86,7 @@ public void configure(Binder binder)
         @ForMetadata
         public static ConnectionFactory createConnectionFactory(H2DatabaseConfig config)
         {
-            String url = "jdbc:h2:" + config.getFilename();
+            String url = format("jdbc:h2:%s;DB_CLOSE_DELAY=-1", config.getFilename());
```

Comment:
Worth documenting why DB_CLOSE_DELAY matters?

Reply:
per h2 docs, `DB_CLOSE_DELAY=-1` will keep database around for longer (which is probably desireable in the tests)
i expect this will address some H2 related failures that i observed in my stress test https://github.com/trinodb/trino/pull/12730
will this also prevent OOM? why?
intuitively `DB_CLOSE_DELAY=-1`  increases memory usage, not decreases.

Reply:
I have no idea. It's been there forever.